utilities/imgcat

```diff
@@ -99,12 +141,11 @@ else
 fi
 
 # Show help if no arguments and no stdin.
-if [ $has_stdin = f ] && [ $# -eq 0 ]; then
+if [ $has_stdin == f ] && [ $# -eq 0 ]; then
```

Comment:
I think `==` is less-preferred; the man page for `test` says it's supported for compatibility with other implementations. I guess it might break on some unixes (and imgcat has to be portable to weird stuff like ancient CentOSes, BSDs, etc.)

Reply:
I did it purely automatically, old-good habit to avoid confusion between comparison and assignment expressions, which is so crucial in some languages.

Do we really need such pure sh-compatibility here? `bash` is explicitly required in all iTerm2 utilities. And there are multiple places exists where `[[` expressions with `==` (pattern-matching) and `=~` (regular expressions) used. So in case we need to be pure sh-compatible we mush change all of them.

BTW for now `[` and `[[` with `==` works even in busybox `ash`.